components/StorySeo/StorySeo.tsx

```diff
@@ -1,15 +1,23 @@
 import type { ExtendedStory } from '@prezly/sdk';
 import { ArticleJsonLd, NextSeo } from 'next-seo';
 
+import escapedQuotedString from '@/utils/escapedQuotedString';
+
 type Props = {
     story: ExtendedStory;
 };
 
+type ManipulatedString = {
+    title: string;
+    description: string;
+};
+
 function StorySeo({ story }: Props) {
     const { title, subtitle, published_at, updated_at, author, oembed, newsroom, summary } = story;
 
     const authorName = author?.display_name || author?.email || 'Unknown';
     const description = subtitle || summary;
+    const modifiedString = escapedQuotedString({ title, description }) as ManipulatedString;
```

Comment:
This is an overkill. It's much easier to make a function that manipulates a string, but also your object would be recreated on each render, resulting in unwanted re-renders.

Reply:
> your object would be recreated on each render, resulting in unwanted re-renders.

That is exactly what I thought before and we could directly change it in the `ArticleJsonLd` with RegEx but I'm not sure if that is a feasible solution or not

Reply:
The simpler solution is always feasible

Reply:
```js
title={JSON.stringify(title).slice(1, -1)}
```
what do you think about the approach above? this also works

Reply:
This is what I've linked to you in the comment below

Reply:
fixed 👍🏼

Reply:
<img width="848" alt="Screenshot 2022-04-11 at 11 37 40" src="https://user-images.githubusercontent.com/69587933/162711106-406e0466-c30b-4413-a261-6958b9cc0e08.png">